<launch>
 <param name="alt_zero" value="169.04000000000002" />
 <param name="lat_zero" value="45.47766945833333" />
 <param name="lon_zero" value="9.226740196666666" />

 <node pkg="first_project" name="first_node" type="gps_to_odom" ></node>

<node pkg="first_project" name="gps_o2t" type="odom_to_tf" >
  <remap to="/gps_odom" from="/input_odom"/>
  <param name="root_frame" value="world" />
  <param name="child_frame" value="gps_odom" />
</node>


 <node pkg="first_project" name="wheel_o2t" type="odom_to_tf" >
  <remap to="/odom" from="/input_odom"/>
  <param name="root_frame" value="world" />
  <param name="child_frame" value="odom" />
</node>


<node pkg="first_project" name="lidar_select" type="lidar_select" ></node>

<node type="rviz" name="rviz" pkg="rviz" ></node> 
</launch>
